demos/keypoints_bounding_boxes/keypoints_bounding_boxes_demo.py

```diff
@@ -0,0 +1,235 @@
+            label = int(detection_as_xyxy[5].item())            
+            norfair_detections.append(
+                Detection(points=bbox, scores=scores, label=label)
+            )
+
+    return norfair_detections
+
+########################################
+
+
+
+
+# Distance function
+def keypoints_distance(detected_pose, tracked_pose):
+    # Use different distances for bounding boxes and keypoints
+    if detected_pose.label != 0:
+        detection_centroid = np.sum(detected_pose.points, axis=0)/len(detected_pose.points)
+        tracked_centroid = np.sum(tracked_pose.estimate, axis=0)/len(detected_pose.points)
+        distances = np.linalg.norm(detection_centroid - tracked_centroid, axis=0)
+        return distances / (KEYPOINT_DIST_THRESHOLD + distances)
+
+    distances = np.linalg.norm(detected_pose.points - tracked_pose.estimate, axis=1)
+    match_num = np.count_nonzero(
+        (distances < KEYPOINT_DIST_THRESHOLD)
+        * (detected_pose.scores > DETECTION_THRESHOLD)
+        * (tracked_pose.last_detection.scores > DETECTION_THRESHOLD)
+    )
+    return 1 / (1 + match_num)
+
+
+if __name__ == "__main__":
+
+    # CLI configuration
+    parser = argparse.ArgumentParser(description="Track objects in a video.")
+    parser.add_argument("files", type=str, nargs="+", help="Video files to process")
+    parser.add_argument("--skip-frame", dest="skip_frame", type=int, default=1, help="Frame skip period")
+    parser.add_argument("--detector_path", type=str, default="yolov5m6.pt", help="YOLOv5 model path")
+    parser.add_argument("--img_size", type=int, default="720", help="YOLOv5 inference size (pixels)")
+    parser.add_argument("--conf_thres", type=float, default="0.25", help="YOLOv5 object confidence threshold")
+    parser.add_argument("--iou_thresh", type=float, default="0.45", help="YOLOv5 IOU threshold for NMS")
+    parser.add_argument('--classes', nargs='+', type=int, help='Filter by class: --classes 0, or --classes 0 2 3')
+    parser.add_argument("--device", type=str, default=None, help="Inference device: 'cpu' or 'cuda'")
+    parser.add_argument("--track_points", type=str, default="centroid", help="Track points: 'centroid' or 'bbox'")
+    args = parser.parse_args()
+
+    # Process Videos
+    detector = OpenposeDetector()
+    datum = op.Datum()
+    model = YOLO(args.detector_path, device=args.device)
+
+    for input_path in args.files:
+        print(f"Video: {input_path}")
+        video = Video(input_path=input_path)
+        tracker = Tracker(
+            distance_function=keypoints_distance,
+            distance_threshold=DISTANCE_THRESHOLD,
+            detection_threshold=DETECTION_THRESHOLD,
+            hit_inertia_min=HIT_INERTIA_MIN,
+            hit_inertia_max=HIT_INERTIA_MAX,
+            initialization_delay=INITIALIZATION_DELAY,
+            point_transience=POINT_TRANSIENCE,
+        )
+        KEYPOINT_DIST_THRESHOLD = video.input_height / 40
+    
+        for i, frame in enumerate(video):
```

Comment:
I'd personally not support skipping frames in this demo. There are a lot of things going on, and we want to make each demo show off a small, easy to grasp, subset of Norfair's features. So anything that makes the demo shorter and simpler is better (except not tracking 2 classes 😂).